quickstarts/aws/aws-smd/config.yml

```diff
@@ -0,0 +1,23 @@
+slug: aws-system-manager-distributor
+description: |
+  ## Install New Relic Infrastructure Agent on a diverse fleet of AWS EC2 instances all at once!
+  ## Get started!
+  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-system-manager-distributor) to learn 
```

Comment:
Hey @4demos - looks like this link doesn't exist or might be incorrect. Would you mind double checking the link?